tfhe/src/integer/gpu/ciphertext/compressed_ciphertext_list.rs

```diff
@@ -158,7 +158,7 @@ mod tests {
         );
 
         let (cuda_compression_key, cuda_decompression_key) =
-            radix_cks.new_cuda_compression_decompression_keys(&private_compression_key, &streams);
+            radix_cks.new_cuda_compression_decompression_keys(&private_compression_key.0, &streams);
```

Comment:
I would move the .0 in the cuda code and make sure this interface uses an integer key

Reply:
Done